src/main/java/wtf/g4s8/examples/spaxos/InMemoryAcceptor.java

```diff
@@ -38,6 +40,9 @@
      * Memory storage.
      */
     private final AtomicReference<T> mem;
+    private final Integer proposerServerId;
+    private final Integer parentServerId;
```

Comment:
@stepanvalyavskiy why not just `int`?

Reply:
can't say now, it was rewritten too many times. will fix